sumomongodbatlascollector/api.py

```diff
@@ -70,12 +78,13 @@ def fetch(self):
                 else:
                     self.log.error(f'''Failed to send LogType: {self.get_key()}''')
             elif fetch_success and len(content) == 0:
-                self.log.info(f'''No results window LogType: {log_type} kwargs: {kwargs} status: {fetch_success}''')
+                self.log.info(f'''No results window LogType: {log_type} kwargs: {kwargs} status: {fetch_success} url: {url}''')
                 is_move_fetch_window, new_state = self.check_move_fetch_window(kwargs)
                 if is_move_fetch_window:
                     self.save_state(**new_state)
                     self.log.debug(f'''Moving fetched window newstate: {new_state}''')
             else:
+                self.log.info(f'''No results window LogType: {log_type} kwargs: {kwargs} status: {fetch_success} url: {url}''')
```

Comment:
else condition is a error condition so we can put url and kwargs in self.log.error statement in line 88

Reply:
Moved to the context manager based logging